pals/core.py

```diff
@@ -116,7 +119,8 @@ def release(self):
         if self.conn is None:
             return False
 
-        sql = sa.text('select pg_advisory_unlock(:lock_num)')
+        shared_suffix = '_shared' if self.shared else ''
```

Comment:
Looks like `shared_suffix` is defined twice the exact same way?

Reply:
Good idea, will fix.